src/views/admin/UserDetailsView.vue

```diff
@@ -15,8 +15,9 @@
 
   const password: Ref<string> = ref('')
 
-  function navigateToUserTable(): void {
-    router.push({ name: 'user-management' })
+  async function navigateToUserTable(): Promise<void> {
+    await router.push({ name: 'user-management' })
+    personStore.errorCode = ''
```

Comment:
I think it would be better to clear the error when fetching a person (e.g. in the `personStore.getPersonById()`-method)

This fixes the bug as it is described in the ticket, but you can still encounter it using other ways (Open invalid user URL -> use the sidebar to navigate back to the user-list -> klick on a valid user)